android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
Can you add android/.settings to the gitignore and remove this file from git please ?